### DISCUSSION TITLE
At what height are the measurements for windSpeed?

### DISCUSSION THREAD

**Discussion by jarataraj** [weather-gov/api#319] [Q&A]
**NOTE**
Please do not report operational issues (such as outages) here. We can not support these issues via this forum. Please report these as described here: https://weather-gov.github.io/api/reporting-issues

**Please describe what you're trying to do.**
A clear and concise description of what your issue is. Ex. I'm looking for forecast data...

Is there a certain height that the measurements for windSpeed are taken at? I am using a function for calculating an approximation of wet bulb globe temperature that requires the 2m wind speed, but can use a helper function to estimate the 2m wind speed given the wind speed at a different height. 

Thank you for your help

**Comment by KTibow** [comment]
Not official, but the NWS uses data from a variety of places. Some places might be higher or lower. I think you might want to try to Google the name of your weather station so you can locate its elevation or some pictures.

**Comment by scadergit** [comment]
The observation endpoint metadata includes the elevation of the station, but to @KTibow 's point they are land-based. So the height is merely the height of the object (building, tower, etc.) that the observation equipment is attached to.

We are working on efforts to add IWXXM observations to the API.

**Comment by joepperkins** [comment]
I’ve found this API from the USGS very accurate, parameters are EPSG:4326 not EPSG:3857<https://en.wikipedia.org/wiki/EPSG_Geodetic_Parameter_Dataset>.

https://nationalmap.gov/epqs/pqs.php?x={Longitude}&y={Latitude}&units=Feet&output=json

Joe Perkins
